src/status_im/contexts/wallet/events.cljs

```diff
@@ -220,12 +220,14 @@
 (rf/reg-event-fx :wallet/get-keypairs get-keypairs)
 
 (rf/reg-event-fx :wallet/bridge-select-token
- (fn [{:keys [db]} [{:keys [token stack-id]}]]
-   (let [to-address (get-in db [:wallet :current-viewing-account-address])]
-     {:db (-> db
-              (assoc-in [:wallet :ui :send :token] token)
-              (assoc-in [:wallet :ui :send :to-address] to-address)
-              (assoc-in [:wallet :ui :send :tx-type] :tx/bridge))
+ (fn [{:keys [db]} [{:keys [token token-symbol stack-id]}]]
+   (let [missing-recipient? (-> db :wallet :ui :send :to-address nil?)
+         to-address         (-> db :wallet :current-viewing-account-address)]
+     {:db (cond-> db
+            :always            (assoc-in [:wallet :ui :send :tx-type] :tx/bridge)
+            token              (assoc-in [:wallet :ui :send :token] token)
+            token-symbol       (assoc-in [:wallet :ui :send :token-symbol] token-symbol)
+            missing-recipient? (assoc-in [:wallet :ui :send :to-address] to-address))
```

Comment:
When we bridge from the home screen, we don't know to which account we will bridge, on the other hand, when we visit an account page we already know it, this change makes this event reusable for both flows